src/touch-region.jsx

```diff
@@ -1,12 +1,12 @@
-import { React, useEffect, useState, createRef } from 'react';
-import PropTypes from 'prop-types';
-import './touch-region.scss';
-import { createGridArea } from 'os2display-grid-generator';
-import Slide from './slide';
-import ErrorBoundary from './error-boundary';
-import idFromPath from './id-from-path';
-import { ReactComponent as IconClose } from './assets/icon-close.svg';
-import { ReactComponent as IconPointer } from './assets/icon-pointer.svg';
+import { React, useEffect, useState, createRef } from "react";
+import PropTypes from "prop-types";
+import "./touch-region.scss";
+import { createGridArea } from "os2display-grid-generator";
+import Slide from "./slide";
+import ErrorBoundary from "./error-boundary";
+import idFromPath from "./id-from-path";
+import IconClose from "./assets/icon-close.svg?react";
```

Comment:
Is this some new way of handling svg components?

Reply:
https://www.npmjs.com/package/vite-plugin-svgr#usage

<img width="761" alt="image" src="https://github.com/aroskanalen/display-client/assets/15377965/d6db60ae-5001-47b4-8e57-1201062fc456">

Yes it is vite-plugin-svgr